### PR TITLE
Nanotrasen Secure Transport ship

### PR DESCRIPTION
**Pull request by BBBBAface**
## About The Pull Request

This is another ship I made for the main branch of Shiptest that I've modified to work better on this server. It is essentially a Nanotrasen equivalent to the Luxembourg, with different pros and cons and a slightly higher price-tag. 

Features:
High-Pressure cycling airlock for extremely fast cycle times
AI core 
Access-restricted storage, armory and bridge
Plasma tank designed to be easily refilled
Launchpad
Moderate firepower for 3 - 5 Security officers
Robust medbay
Large hallway for ease of mobility and customizability potential
Medium-sized bar

## Why It's Good For The Game

More ships are good; More Nanotrasen ships are also good.

## Changelog
:cl:
add: SHIP
/:cl:

### CODE REVIEW

**Comment by BBBBAface** [comment]
ignore all those commits they're a side effect of me being bad at using github

File: _maps/configs/NT-C_clipper.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NT-C",
+	"namelists": ["GENERAL", "BRITISH_NAVY", "MERCANTILE", "SPACE"],
+	"map_name": "Indispubable-class Secure Transport",
```
Comment (GuiltyNeko) [inline _maps/configs/NT-C_clipper.json]: Fix this

Comment (BBBBAface) [reply _maps/configs/NT-C_clipper.json]: fixed the typo, anything else?

Comment (GuiltyNeko) [reply _maps/configs/NT-C_clipper.json]: I haven't given it a proper look-through yet. Check your decals, make sure the ship is capable of docking, undocking, and moving on the overmap. And please, please, don't run piping through walls, it makes it significantly harder to fix in-game.

**Review by GuiltyNeko — CHANGES_REQUESTED** [review]
Port hallway has 2 APCs, don't run pipes and cables under walls.

**Comment by GuiltyNeko** [comment]
I'll testmerge this, but as a general rule of thumb run pipes and cables on the same tile, space out the scrubbers and vents in a room from each other, run pipes and cables as close to the middle of the hall as you can (minus glass floors) and make the pipe as straight as possible, to make it as simple as possible to find the pipes and cables without a T-ray scanner and to just generally make it a much simpler layout.

**Pull request closed without merging**